styles.css

```diff
@@ -42,4 +41,32 @@ body {
   padding: 0;
   font-family: inherit;
   font-size: 20px;
+  font-weight: bold;
+  color: inherit;
+  border-radius: 0;
+  border: 0;
+  border-left: 1px solid var(--secondaryColor);
+  border-top: 1px solid var(--secondaryColor);
+  cursor: pointer;
+}
+
+.button:nth-child(4n) {
+  border-right: 1px solid var(--secondaryColor);
+  background-color: var(--operButtonsColor);
+}
+
+.button:nth-child(n + 21) {
```

Comment:
`nth-child` with a weird expression looks unclear. Just create another class and add it to elements